yjit/src/cruby.rs

```diff
@@ -356,6 +425,13 @@ pub const VM_ENV_DATA_INDEX_FLAGS:isize = 0;
 pub const VM_ENV_DATA_SIZE:usize = 3;
 pub const VM_ENV_FLAG_WB_REQUIRED:usize = 0x008;
 
+// From vm_callinfo.h
+pub const VM_CALL_ARGS_SPLAT:u32    = 1 << 0;
+pub const VM_CALL_ARGS_BLOCKARG:u32 = 1 << 1;
+pub const VM_CALL_FCALL:u32         = 1 << 2;
+pub const VM_CALL_KWARG:u32         = 1 << 6;
+pub const VM_CALL_KW_SPLAT:u32      = 1 << 7;
```

Comment:
These didn't get successfully auto-exported?

Reply:
It does export defines, at least simple ones.

Reply:
It's quite possible that this is just too advanced for bindgen.

Reply:
I think we could export `enum vm_call_flag_bits` and hand write the consts them as `1 << VM_*_bit` that way we are notified when the C changes.

Reply:
It would appear so. So I think we have to include them explicitly.

Reply:
Okay. We can do it the way Alan suggests (import vm_call_flag_bits, use that in our constants.) I've updated the PR.